aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSampler.java

```diff
@@ -148,8 +149,8 @@ private void getAndUpdateSampler() {
   }
 
   private void scheduleSamplerUpdate() {
-    long delay = pollingIntervalNanos + RANDOM.nextInt(jitterNanos);
-    pollFuture = executor.schedule(this::getAndUpdateSampler, delay, TimeUnit.NANOSECONDS);
+    long delay = pollingIntervalMillis + RANDOM.nextInt(jitterMillis);
```

Comment:
Can we stick with nanos and switch to `Random.longs(0, jitterNanos)` instead of switching to millis? It's nice to avoid dropping precision mysteriously when possible, even in practice it generally shouldn't matter here.

Reply:
Good point! This requires significantly less changes. I pushed an updated commit.